nomad/structs/csi.go

```diff
@@ -6,12 +6,13 @@ package structs
 import (
 	"errors"
 	"fmt"
-	"maps"
-	"slices"
 	"strings"
 	"time"
 
 	multierror "github.com/hashicorp/go-multierror"
+	"golang.org/x/exp/maps"
+	"golang.org/x/exp/slices"
```

Comment:
I think these are in the stdlib now and there was a PR that got merged removing their use everywhere recently.

Reply:
ah I fell down on the wrong side of a rebase I think